fccloudapi.php

```diff
@@ -2254,6 +2265,81 @@ protected function parseHeader($result)
         return $buffer;
 	}
 
+    protected function queryToArray($queryString)
```

Comment:
This function looks good but isn't parse_str funtion doing exactly that?

Reply:
Good point! Almost 15 years of PHP and I didn't know about that function... lol

And it's not new for sure, because of the ugly reference parameter. Looks like old-ugly PHP implementations...

It's fixed.

Reply:
I didn't know that function TBH but I couldn't believe that there is no built-in method for something like that :)